src/main/java/uk/gov/hmcts/befta/player/BackEndFunctionalTestScenarioContext.java

```diff
@@ -23,7 +23,7 @@ public class BackEndFunctionalTestScenarioContext {
 
     private static final String[] TEST_DATA_RESOURCE_PACKAGES = { "features" };
 
-    static final HttpTestDataSource DATA_SOURCE = HttpTestDataSourceFactory.createHttpTestDataSource(TEST_DATA_RESOURCE_PACKAGES);
+    static HttpTestDataSource DATA_SOURCE = HttpTestDataSourceFactory.createHttpTestDataSource(TEST_DATA_RESOURCE_PACKAGES);
```

Comment:
With Java 21 optimisations, I believe `final` makes this inlined at compile time and makes mocking via reflection impossible.